### PR TITLE
Test app UI updated

### PR DESCRIPTION
**Pull request by olmisa**
* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

The puppet application UI was updated for 'disableAutomaticCheckForUpdate' and 'checkForUpdate' methods.

## Related PRs or issues

[AB#75365](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/75365)

### CODE REVIEW

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml [hunk 1]
```diff
@@ -6,6 +6,10 @@
             <ViewCell>
                 <Picker x:Name="UpdateTrackPicker" Title="Update track" PropertyChanged="ChangeUpdateTrack" />
             </ViewCell>
+            <SwitchCell Text="Automatic update check" x:Name="AutomaticUpdateCheckSwitchCell" OnChanged="OnAutomaticUpdateCheckChanged" />
+            <ViewCell>
+                <Button Text="Check updates" Clicked="CheckForUpdateClicked"/>
```
Comment (annakocheshkova) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml]:
```suggestion
                <Button Text="Check for updates" Clicked="CheckForUpdateClicked"/>
```

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml [hunk 2]
```diff
@@ -6,6 +6,10 @@
             <ViewCell>
                 <Picker x:Name="UpdateTrackPicker" Title="Update track" PropertyChanged="ChangeUpdateTrack" />
             </ViewCell>
+            <SwitchCell Text="Automatic update check" x:Name="AutomaticUpdateCheckSwitchCell" OnChanged="OnAutomaticUpdateCheckChanged" />
```
Comment (annakocheshkova) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml]:
```suggestion
            <SwitchCell Text="Automatic check for update" x:Name="AutomaticUpdateCheckSwitchCell" OnChanged="OnAutomaticUpdateCheckChanged" />
```

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/App.xaml.cs [hunk 3]
```diff
@@ -79,6 +79,10 @@ protected override void OnStart()
                 {
                     Distribute.UpdateTrack = updateTrack.Value;
                 }
+                if (Application.Current.Properties.TryGetValue(Constants.AutomaticUpdateCheckKey, out object persistedObject) && !(bool)persistedObject)
```
Comment (guperrot) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/App.xaml.cs]: Screen Shot 2020-02-20 at 5 39 22 PM

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml.cs [hunk 4]
```diff
@@ -86,14 +85,16 @@ async void UpdatePushEnabled(object sender, ToggledEventArgs e)
 
         async void RefreshDistributeEnabled(bool _appCenterEnabled)
         {
-            DistributeEnabledSwitchCell.On = await Distribute.IsEnabledAsync();
-            DistributeEnabledSwitchCell.IsEnabled = _appCenterEnabled;
-            RefreshDistributeTrackUpdate();
+            var distributeEnabled = await Distribute.IsEnabledAsync();
+            DistributeEnabledSwitchCell.On = distributeEnabled;
+            DistributeEnabledSwitchCell.IsEnabled = _appCenterEnabled && distributeEnabled;
```
Comment (guperrot) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml.cs]:
```suggestion
            DistributeEnabledSwitchCell.IsEnabled = _appCenterEnabled;
```

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml.cs [hunk 5]
```diff
@@ -86,14 +85,16 @@ async void UpdatePushEnabled(object sender, ToggledEventArgs e)
...
+            RefreshDistributeTrackUpdate(distributeEnabled);
+            RefreshAutomaticUpdateCheck(_appCenterEnabled && distributeEnabled);
         }
 
-        async void RefreshDistributeTrackUpdate()
+        async void RefreshDistributeTrackUpdate(bool? _distributeEnabled = null)
         {
-            var isDistributeEnable = await Distribute.IsEnabledAsync();
+            var isDistributeEnable = _distributeEnabled.HasValue ? _distributeEnabled.Value : await Distribute.IsEnabledAsync();
```
Comment (guperrot) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml.cs]: Update track state does not depend on distribute enabled state, we should revert those changes.

**Pull request merged**